copter/source/docs/traditional-helicopter-rsc-setup.rst

```diff
@@ -0,0 +1,19 @@
+.. _traditional-helicopter-rsc-setup:
+
+===============================
+Rotor Speed Control Setup
+===============================
+
+Rotor speed controller set up is described here in video 
+3 for electric powered helicopters and video 4 for piston 
+powered helicopters. 
+
+traditional-helicopter-configuration.rst
+
```

Comment:
:ref:`traditional-helicopter-configuration.rst`

Reply:
you cant see the formatting above since it shows different here    back ticks surround the tex link